### PR TITLE
Upload to S3 only if local file changed

### PR DESCRIPTION
**Pull request by philipmw**
On slow connections with large JARs, this speeds up the development cycle.

### CODE REVIEW

**Comment by philipmw** [comment]
What do you think, @SeanRoy?

Also, I am happy to help you maintain this package, if you wanna give me commit access.

**Comment by SeanRoy** [comment]
Thank Phil, I'll review your PR and test it over the next few days.

File: pom.xml [hunk 1]
```diff
@@ -24,6 +24,10 @@
       <email>Sean.Roy@gmail.com</email>
       <organizationUrl>https://github.com/SeanRoy/lambduh-maven-plugin</organizationUrl>
     </developer>
+    <developer>
+      <name>Philip M. White</name>
+      <email>philip@mailworks.org</email>
+    </developer>
   </developers>
```
Comment (SeanRoy) [inline pom.xml]: Please move yourself to a contributors section.  I won't be adding other committers at this time.

 ```
<contributors>
  <contributor>
    <name>Philip M. White</name>
    <email>philip@mailworks.org</email>
  <contributor>
</contributors>
 ```

Comment (philipmw) [inline pom.xml]: Sure thing - I didn't know about the distinction.

**Comment by SeanRoy** [comment]
Please re-open this pull request against 1.0.4-SNAPSHOT

File: pom.xml (hunk 1 above)
Comment (SeanRoy) [inline pom.xml]: No worries, thanks for your contribution!

**Comment by barbeau** [comment]
+1 for this feature

**Comment by philipmw** [comment]
Closing this PR so I can reopen it against another branch.

**Pull request closed without merging**